code/modules/fishing/fish/types/rift.dm

```diff
@@ -400,3 +400,427 @@
+
+	if(!locate(src) in user)
+		return ..()
+
+	if((user.usable_hands < 2) && !HAS_TRAIT(user, TRAIT_STRENGTH))
+		to_chat(user, span_notice("[src] is too dense to twist apart with only one hand."))
+		return
+
+	to_chat(user, span_danger("You start pulling and twisting [src], trying to split it down the middle..."))
+	if(!do_after(user, 5 SECONDS, src))
+		return
+
+	playsound(get_turf(user), 'sound/effects/wounds/crack1.ogg', 60)
+	set_status(FISH_DEAD)
+	var/cracked = new /obj/item/organ/ears/babbelfish(user)
+	user.put_in_hands(cracked)
+	qdel(src)
+
+/** These ears grant amazing and supernatural hearing, but they also screw over your knowledge of language.
+ * Three outcomes:
+ * The user is able to speak all languages, but can't understand any. Even their own words come across as unintelligible gibberish.
+ * The user is able to understand all languages, but can't speak any. They can only 'make strange noises'.
+ * The user is able to understand and speak all languages. Rare.
+ * All this to be a curator, this is silly.
+*/
+/obj/item/organ/ears/babbelfish
+	name = "split babbelfish halves"
+	icon_state = "babbearfish"
+	desc = "Both halves of a babbelfish after being twisted apart. The legends claim inserting these can unlock your psychic potential. It probably wasn't worth hearing that wail, though."
+	organ_traits = list(TRAIT_XRAY_HEARING, TRAIT_GOOD_HEARING)
+
+	healing_factor = STANDARD_ORGAN_HEALING * 5
+	decay_factor = 0
+
+	low_threshold_passed = span_noticealien("Psychic whispers make it a bit difficult to hear sometimes..")
+	now_failing = span_noticealien("Psychic noise is overcrowding your senses!")
+	now_fixed = span_noticealien("The psychic noise starts to fade.")
+	low_threshold_cleared = span_noticealien("The whispers leave you alone.")
+
+	bang_protect = 5
+	damage_multiplier = 0.1
+	visual = TRUE
+	/// Overlay for the mob sprite because actual organ overlays are a fucking unusable nightmare
+	var/datum/bodypart_overlay/simple/babbearfish/babbel_overlay
+	var/bound_component
+	var/datum/language_holder/removal_holder
+
+/**
+ * The bodypart overlay for babbearfish.
+ * We don't need anything other than this icon (and to hide it sometimes), so it being mutant is unnecessary and a waste of space..
+ * Which breaks everything else. So we need to make it simple. Which organs don't innately support, so we need to just haphazardly slap it on.
+ */
+/datum/bodypart_overlay/simple/babbearfish
+	icon_state = "babbearfish"
+
+/datum/bodypart_overlay/simple/babbearfish/can_draw_on_bodypart(mob/living/carbon/human/human)
+	if((human.head?.flags_inv & HIDEEARS) || (human.wear_mask?.flags_inv & HIDEEARS))
+		return FALSE
+	return TRUE
+
+/obj/item/organ/ears/babbelfish/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/noticable_organ, "%PRONOUN_They %PRONOUN_have weird, deep-rooted obsidian tubes sticking out of where their ears should be.")
+	AddElement(/datum/element/update_icon_blocker) // wats this even do
```

Comment:
It stops the icons of the object (not the mob it's inserted into) from updating. Why would a set of ears that doesn't have other states or overlays that I recall need this anyway?

Reply:
the cockroach wings organ had it for some erason

it's meant to have overlays but it dosent work.
```suggestion
```